api/drone/drone.proto

```diff
@@ -7,24 +7,66 @@ syntax = "proto3";
 package openpitrix.drone;
 
 service DroneService {
-	rpc StartConfd (Task) returns (Empty);
+	rpc GetInfo(Empty) returns (Info);
+	rpc GetConfdConfig(Empty) returns (ConfdConfig);
+	rpc GetBackendConfig(Empty) returns (BackendConfig);
+	rpc StartConfd (StartConfdRequest) returns (Empty);
+	rpc StopConfd (StopConfdRequest) returns (Empty);
+	rpc GetConfdStatus(Empty) returns (ConfdStatus)
 }
 
-message Task {
-	string id = 1;
-	string action = 2;
-	string target = 3;
-
-	TaskDirective directive = 4;
+service DroneServiceForFrontGate {
+	rpc GetRegisterCmdStatus (RegisterCmd) returns (ConfdStatus);
 }
 
-message TaskDirective {
+message Info {
 	string drone_ip = 1;
-	string frontgate_id = 2;
+	repeated string cmds = 2;
+}
+
+message ConfdConfig {
+	// libconfd.Config
+	google.protobuf.StringValue conf_dir = 1;
+	google.protobuf.Int32Value interval = 2;
+	google.protobuf.BoolValue noop =3;
+	google.protobuf.StringValue prefix = 4;
+	google.protobuf.BoolValue sync_only = 5;
+	google.protobuf.StringValue log_level = 6;
+}
+
+message BackendConfig {
+	google.protobuf.StringValue type = 1;
+	repeated string hosts = 2;
+	google.protobuf.StringValue username = 3;
+	google.protobuf.StringValue password = 4;
+	google.protobuf.StringValue client_ca_keys = 5;
+	google.protobuf.StringValue client_cert = 6;
+	google.protobuf.StringValue client_key = 7;
+
+}
+
+message StartConfdRequest {
+	ConfdConfig confd_config = 1;
+	BackendConfig backend_config = 2;
+}
+
+message StopConfdRequest {
+	ConfdConfig confd_config = 1;
+	BackendConfig backend_config = 2;
+}
+
+message ConfdStatus {
+	string status = 1;
+}
+
+message RegisterCmd {
```

Comment:
Can this be changed to `GetRegisterCmdStatusRequest`?
Only cmd_id is needed, cmd_id can be subtask_id. 
```
message GetRegisterCmdStatusRequest {
  string cmd_id =1;
}
```